synfig-studio/src/gui/canvasview.cpp

```diff
@@ -2928,6 +2929,23 @@ remove_waypoints(std::set<Waypoint, std::less<UniqueID> > waypoints,
 	}
 }
 
+
+void
+CanvasView::copy_waypoints(synfig::Time time)
+{
+	current_widget_timetrack->handle_copied_waypoints_selection();
+	copied_waypoint_time = time;
+	current_widget_timetrack->waypoint_mouse_copy=true;
```

Comment:
I'm still wondering if we should create some kind of waypoint clipboard.
What do you think, @ice0 ?